Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM rust:slim-bookworm AS builder
 
-RUN apt-get update -y && \
-  apt-get install -y make g++ libssl-dev && \
-  rustup target add x86_64-unknown-linux-gnu
+ARG USE_GH_RELEASE=false
+RUN apt-get update -y && apt-get install -y make g++ libssl-dev curl jq tar gzip
 
 WORKDIR /app
 COPY . .
 
-RUN cargo build --release --target x86_64-unknown-linux-gnu
-
+RUN if [ "${USE_GH_RELEASE}" = "true" ]; then \
+    mkdir -p target/$(uname -m)-unknown-linux-gnu/release && \
+    export ZOLA_VERSION=$(curl -sL https://api.github.com/repos/getzola/zola/releases/latest | jq -r .name) && \
```

Comment:
This is not reproducible, can you pass the version to the Dockerfile instead?

Reply:
Totally agree. Just wanted to check if this approach is fine before going all the way.  
I would pass an extra arg like `ZOLA_VERSION` with. I would set the default to latest and override in CI to the current tag. To enforce reproducibility we could also just fail if `USE_GH_RELEASE` is set but `ZOLA_VERSION` is not. Any takes on that?

Reply:
Sounds good